source/src/ModuleApplication.cc

```diff
@@ -406,30 +485,114 @@ namespace dqm4hep {
     
     //-------------------------------------------------------------------------------------------------
     
+    void ModuleApplication::configureEventReader(core::TiXmlElement *element) {
+      if(EVENT_READER != appRunningMode()) {
```

Comment:
OK, you can discard my previous comment about calling `configureEventReader` only if module is an `event_reader` =)